pkg/user/password_reset.go

```diff
@@ -0,0 +1,27 @@
+package user
+
+import (
+	"time"
+
+	"github.com/dgryski/trifles/uuid"
+)
+
+type PasswordReset struct {
+	Token     string
+	CreatedAt time.Time
+}
+
+func NewPasswordReset() *PasswordReset {
+	return &PasswordReset{
+		Token:     generateToken(),
+		CreatedAt: time.Now(),
+	}
+}
+
+func generateToken() string {
+	return uuid.UUIDv4()
+}
+
+func (pr *PasswordReset) IsValidRequest(token string) bool {
```

Comment:
```suggestion
func (pr *PasswordReset) Valid(token string) bool {
```

Short method name will be liked by Go developers